vignettes/tidy-evaluation.Rmd

```diff
@@ -115,7 +115,7 @@ positions. Therefore, `select()` understands column symbols like
 dplyr::select(mtcars, cyl)
 
 # Taking an unquoted symbol:
-var <- quote(cyl)
+var <- quo(cyl)
```

Comment:
This is the only non-typo change. I assume this should be `rlang: quo()` but I could be wrong?